pori_python/graphkb/util.py

```diff
@@ -328,18 +328,6 @@ def get_rid(conn: GraphKBConnection, target: str, name: str) -> str:
     return result[0]["@rid"]
 
 
-def ontologyTermRepr(term: Union[OntologyTerm, str]) -> str:
```

Comment:
is this not used anywhere?

Reply:
I could not find any sign of it anymore.
Linting should pick it up, if it was used witin pori_python.

There are other functions doing effectively the same thing, so it might have been refactored out.  Or was the intended method at one point.